internal/databases/mongo/oplog/applier.go

```diff
@@ -123,13 +123,13 @@ func (ap *DBApplier) Close(ctx context.Context) error {
 	return nil
 }
 
-func (ap *DBApplier) shouldSkip(op, ns string) error {
-	if op == "n" {
+func (ap *DBApplier) shouldSkip(oplog *db.Oplog) error {
+	if oplog.Namespace == "n" {
 		return fmt.Errorf("noop op")
 	}
 
-	// sharded clusters are not supported yet
-	if strings.HasPrefix(ns, "config.") {
+	//sharded clusters are not supported yet
+	if !isOpAllowed(oplog) {
 		return fmt.Errorf("config database op")
```

Comment:
Need more specific message, may be isOpAllowed(oplog) should return such message.
We need it to being able to understand, why some operation was skipped during debug

Reply:
It seems to me that this is not necessary, since here we briefly limit the `config.` collections according to the white list . And giving a detail message for each check is too detailed information that will never be useful to us